modules/wpcom-block-editor/class-jetpack-wpcom-block-editor.php

```diff
@@ -215,7 +215,9 @@ public function enqueue_scripts() {
 			'wpcomGutenberg',
 			array(
 				'switchToClassic' => array(
-					'isVisible' => false,
+					'isVisible' => $is_calypsoify,
+					'label'     => __( 'Switch to Classic Editor', 'jetpack' ),
+					'url'       => Jetpack_Calypsoify::getInstance()->get_switch_to_classic_editor_url(),
```

Comment:
Can anyone see why a fresh call to `getInstance()` is necessary, rather than using the `$Jetpack_Calypsoify` global? Maybe something to do with sequence, but Calypsoify seems to be executed before `Jetpack_WPCOM_Block_Editor`.

Reply:
This isn't critical; we can follow up when we attempt to align the JP and dotcom Calypsoify classes at a  later date.